packages/cli/test/mocks/client.ts

```diff
@@ -109,6 +122,6 @@ beforeEach(() => {
   client.reset();
 });
 
-afterAll(() => {
-  client.stopMockServer();
+afterAll(async () => {
+  await client.stopMockServer();
```

Comment:
This works as written before, but the mock server was still listening after the test suite completed for some amount of time. When I was using `yarn test --detectOpenHandles`, this mock server listener was listed as an open handle when it's not realllly going to stay open for much longer after the test runner completes. It caused me to spend time digging into this when my real problem was something else.

This change makes the test suite wait for the mock server to be shut down.

Reply:
Nice find!